### PR TITLE
Bump Terraform and Packer versions. (and upstream changes)

### PR DESCRIPTION
**Pull request by felddy**
# 

## 🗣 Description
Upstream changes.

Bump Terraform and Packer to latest releases.  See:
* https://github.com/hashicorp/terraform/blob/v0.12/CHANGELOG.md#01228-june-25-2020
* https://github.com/hashicorp/packer/blob/master/CHANGELOG.md#160-june-09-2020

## 💭 Motivation and Context

Keeping current.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -4,4 +4,4 @@
 # the repo. Unless a later match takes precedence,
 # these owners will be requested for review when someone
 # opens a pull request.
-*       @dav3r @felddy @jsf9k @mcdonnnj @cisagov/team-ois
+*       @dav3r @felddy @hillaryj @jsf9k @mcdonnnj @cisagov/team-ois
```
Comment (jsf9k) [inline .github/CODEOWNERS]: There is no point in continuing to include @cisagov/team-ois.  The intent is to list only those devs that are familiar with this repository.

**Review by jsf9k — CHANGES_REQUESTED** [review]
I have one minor request, but otherwise this looks good to me.

CRITICAL PATH!

**Review by mcdonnnj — APPROVED** [review]
Makes sense ✨

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
Excelsior! 📦

**Pull request merged**